web/docs/tutorials/todo-app/auth.md

```diff
@@ -161,7 +161,7 @@ Now, we can again run
 wasp start
 ```
 
-Try going to `/` in our web app -> it will now ask you to log in, and if you follow the link, you will end up at `/login`.
+Try going to `/` in our web app. It will now redirect you to `/login`, where you'll be asked to authenticate.
```

Comment:
It seems you no longer need to follow any link. When I followed the tutorial, I found that I was automatically redirected to `/login`. So I updated this line accordingly.👍